qiita_core/util.py

```diff
@@ -93,3 +102,26 @@ def wrapper(*args, **kwargs):
         with TRN:
             return func(*args, **kwargs)
     return wrapper
+
+
+def get_qiita_version():
+    """Returns the Qiita version and Git sha if present
+
+    Returns
+    ------
+    tuple (version, sha)
+        The Qiita version and SHA. SHA can be an empty string.
+    """
+    # the actual repo is the abspath of the current file without
```

Comment:
That makes sense.

I didn't catch this last night (sorry) but the code in here can be cleaned up like this:
```python
git_repo_path = dirname(dirname(__file__))
```
which is probably safer in case that `qiita_core` changes.